gapic/packaging/dependencies.yaml

```diff
@@ -91,6 +91,9 @@ google-iam-v1_version:
   ruby:
     name_override: grpc-google-iam
     lower: '0.6.8'
+  java:
+    name_override: grpc-google-iam-v1
```

Comment:
Why would it be risky? If we have to override for every package, then every new package requires this manual work, and people could make mistakes with it. You need to add this same override for every service, right?